packages/scene-composer/src/assets/auto-gen/icons/Camera.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import { SVGProps } from 'react';
 
 const SvgCamera = (props: SVGProps<SVGSVGElement>) => (
+  /* istanbul ignore next */
```

Comment:
Looks like these are causing problems in the build or tests :/

Reply:
Fixed.